app/initializers/irene.js

```diff
@@ -3,137 +3,188 @@ import installPendo from 'irene/utils/install-pendo';
 import installHotjar from 'irene/utils/install-hotjar';
 import customerSuccessBox from 'irene/utils/customer-success-box';
 
+/**
+ * Class to handle environment variables
+ */
 class ENVHandler {
+  /**
+   * @param {Object} envHandlerConst - The environment handler constants
+   */
   constructor(envHandlerConst) {
     this.envHandlerConst = envHandlerConst;
   }
 
+  /**
+   * Checks if runtime configuration is available
+   * @returns {boolean} True if runtime configuration is available
+   */
   isRuntimeAvailable() {
-    return !(typeof runtimeGlobalConfig == 'undefined');
+    return typeof runtimeGlobalConfig !== 'undefined';
   }
 
-  getEnv(env_key) {
-    const host_key = 'IRENE_API_HOST';
-    this.assertEnvKey(env_key);
-    if (this.isAvailableInRuntimeENV(env_key)) {
-      const runtimeValue = this.getRuntimeObject()[env_key];
-      if (env_key === host_key) {
-        if (runtimeValue === '/') {
-          return '';
-        }
+  /**
+   * Gets the value of an environment variable
+   * @param {string} envKey - The key of the environment variable
+   * @returns {string} The value of the environment variable
+   */
+  getEnv(envKey) {
+    const hostKey = 'IRENE_API_HOST';
+    this.assertEnvKey(envKey);
+
+    if (this.isAvailableInRuntimeENV(envKey)) {
+      const runtimeValue = this.getRuntimeObject()[envKey];
+      if (envKey === hostKey && runtimeValue === '/') {
+        return '';
       }
       return runtimeValue;
```

Comment:
Add space above and other return statements below.

Reply:
done